src/overlays/actors/ovl_En_Zl3/z_en_zl3.c

```diff
@@ -2565,8 +2564,7 @@ void func_80B59A80(EnZl3* this, PlayState* play) {
 }
 
 void func_80B59AD0(EnZl3* this, PlayState* play) {
-    // todo look into
-    Actor* thisx = &this->actor; // unused, necessary to use 'this' first to fix regalloc
+    if (this) {}
 
```

Comment:
this function taking a base actor doesnt really make sense, unless all the other functions that lead up to this can.
its possible that they copy pasted the signature from a main function i guess??

it is interestingly the only function that gets called with both `this` and `play` in `func_80B55444`. that fact doesnt really mean much, but its interesting